ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/router_service_insertion.schema.yml

```diff
@@ -8,7 +8,47 @@ type: dict
 keys:
   router_service_insertion:
     type: dict
-    description: Configure network services inserted to data forwarding
+    description: Configure network services inserted to data forwarding.
     keys:
       enabled:
         type: bool
+      connections:
+        type: list
+        primary_key: name
+        items:
+          type: dict
+          keys:
+            name:
+              type: str
+              description: Connection name.
+            ethernet_interface:
+              type: dict
+              description: |-
+                Outgoing physical interface or subinterface to use for the connection.
+                If both ethernet interface and tunnel interface are configured, ethernet interface will be used.
+              keys:
+                name:
+                  type: str
+                  required: true
+                  description: e.g. Ethernet2 or Ethernet2/2.2
+                next_hop:
+                  type: str
+                  required: true
+                  description: Next-hop IPv4 address (without mask).
+            tunnel_interface:
+              type: dict
+              description: |-
+                Outgoing tunnel interface(s) to use for this connection.
+                If both ethernet interface and tunnel interface are configured, ethernet interface will be used.
+              keys:
+                primary:
+                  type: str
+                  description: e.g. Tunnel2
+                secondary:
+                  type: str
+                  description: e.g Tunnel3
+            monitor_connectivity_host:
```

Comment:
It is not clear to me if this is the name of the host defined under monitor connectivity or the actual hostname / ip of the host?

Reply:
it is the name of the host defined under the `monitor connectivity`. This is mentioned in the description below. Maybe it can be reworded?

Reply:
updated